src/slate/classes.py

```diff
@@ -22,7 +22,10 @@
     from pdfminer.pdfparser import PDFPage
 except ImportError:
     from pdfminer.pdfpage import PDFPage
-import utils
+if PYTHON_3:
+    import slate.utils as utils
+else:
+    import utils
```

Comment:
This is a bad way to fix the problem. Just use `from . import utils`, which works on both versions of python